src/app/components/bitcoin-custom-fee/bitcoin-custom-fee-fiat.tsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect } from 'react';
+
+import { Text } from '@stacks/ui';
+import { useField } from 'formik';
+
+import { useBitcoinCustomFee } from './use-bitcoin-custom-fee';
+
+interface BitcoinCustomFeeFiatProps {
+  recipient: string;
+  amount: number;
+  setFeeValue: (value: number) => void;
+}
+
+export function BitcoinCustomFeeFiat({
+  recipient,
+  amount,
+  setFeeValue,
+}: BitcoinCustomFeeFiatProps) {
+  const [field] = useField('feeRate');
+  const { fiatFeeValue, fee } = useBitcoinCustomFee({ feeRate: field.value, amount, recipient });
+
+  useEffect(() => {
+    setFeeValue(fee);
+  }, [fee, setFeeValue]);
```

Comment:
Find this logic a bit confusing. Why do we need to update state this way, by creating an effect to fire value up.  Why can't the parent also use `useBitcoinCustomFee`?